### PR TITLE
add sql 4 assignments files

### PR DESCRIPTION
**Pull request by ghost**
Added assignment 1, assignment 2, assignment 3, and assignment 4 solution files of SQL.

### CODE REVIEW

File: sql_assignment1_create_tables.sql [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+USE student_db;
+
+/******create table school*****/
+create table school(
+    id int PRIMARy KEY,
+    name varchar(20),
+    address varchar(50),
+    principal varchar (20),
+    phone int
+
```
Comment (mugdhajoglekar) [inline sql_assignment1_create_tables.sql]: nit : format the code. remove extra line.

File: sql_assignment1_create_tables.sql [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+/******switch to database*****/
+USE student_db;
+
+/******create table school*****/
+create table school(
+    id int PRIMARy KEY,
+    name varchar(20),
+    address varchar(50),
+    principal varchar (20),
+    phone int
```
Comment (mugdhajoglekar) [inline sql_assignment1_create_tables.sql:13]: try modifying this field with varchar data type. Commit the required queries to modify it

File: sql_assignment1_create_tables.sql [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    'BSL',
+    92,
+    'kumar@gmail',
+    21,
+    5
+);
+
+
+
+SELECT * from student;
```
Comment (mugdhajoglekar) [inline sql_assignment1_create_tables.sql]: It is recommended to use TableName.ColumnName1, TableName.ColumnName2 for select queries.

File: sql_assignment3_queries.sql [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+NOT (odate = '1990-10-03'
+AND cnum > 2003));
+
+
+
+/*What will be the output of the following query?
+Select * from Orders
+where NOT ((odate = ‘1990-10-03’ OR snum >1006) AND amt >= 1500)*/
+Select * from Orders
+where NOT ((odate = '1990-10-03' OR snum >1006) AND amt >= 1500)
```
Comment (mugdhajoglekar) [inline sql_assignment3_queries.sql]: what is the output of this query ?

File: sql_assignment1_create_tables.sql (hunk 2 above)
Comment (ghost) [reply sql_assignment1_create_tables.sql:13]: Updated the phone datatype by using ALTER TABLE at the end of the file.

File: sql_assignment1_create_tables.sql (hunk 3 above)
Comment (ghost) [reply sql_assignment1_create_tables.sql]: Modified the query by using the column name.

File: sql_assignment3_queries.sql (hunk 4 above)
Comment (ghost) [reply sql_assignment3_queries.sql]: Added output result

File: sql_assignment4_queries.sql [hunk 5]
```diff
@@ -0,0 +1,340 @@
...
+
+/*Write a query to get unique department ID from employee table.*/
+select distinct(DEPARTMENT_ID) from employees;
+
+/*Write a query to get all employee details from the employee table order by first name, descending.*/
+select * from employees order by FIRST_NAME DESC;
+
+/*Write a query to get the employee ID, names (first_name, last_name), salary in ascending order of
+salary.*/
+select EMPLOYEE_ID, FIRST_NAME, LAST_NAME, SALARY from employees order by salary;
```
Comment (mugdhajoglekar) [inline sql_assignment4_queries.sql]: The question mentions names (first_name, last_name). Can you try modifying the query ?

File: sql_assignment4_queries.sql [hunk 6]
```diff
@@ -0,0 +1,340 @@
...
+select FIRST_NAME, HIRE_DATE, JOB_ID from employees where JOB_ID ='SA_REP' OR JOB_ID= 'IT_PROG';
+
+/*Display details of employee with ID 150 or 160.*/
+select * from employees where EMPLOYEE_ID = 150 or EMPLOYEE_ID = 160;
+
+/*Display first name, salary, commission pct, and hire date for employees with salary less than 10000.*/
+select FIRST_NAME, SALARY, COMMISSION_PCT, HIRE_DATE from employees where salary < 10000;
+
+/*Display employees where the first name or last name starts with S.*/
+select * from employees where FIRST_NAME LIKE 'A%' or LAST_NAME LIKE 'S%';
```
Comment (mugdhajoglekar) [inline sql_assignment4_queries.sql]: A% ?

File: sql_assignment4_queries.sql (hunk 5 above)
Comment (ghost) [reply sql_assignment4_queries.sql]: Removed employee_id from query

File: sql_assignment4_queries.sql (hunk 6 above)
Comment (ghost) [reply sql_assignment4_queries.sql]: Replaced A% by S%

**Pull request closed without merging**